Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -82,6 +102,7 @@ def StateActivity(self, state, mouse1, mouse2):
             self.cc_cnt += 1
             mouse1.DeliverReward(Locations.Cooperate, self.reward_time)
             mouse2.DeliverReward(Locations.Cooperate, self.reward_time)
+            self.time_to_make_decision = time.time() - self.time_start
```

Comment:
I think this line not needed here. Should only appear in the decision states.

Reply:
actually i think its best if you ignore this PR .too.i will close it .the version that works for sure with the time stamps is in the data analysis changes PR